setup.py

```diff
@@ -16,7 +16,7 @@
     include_package_data=True,
     install_requires=[
         "playwright>=1.18",
-        "pytest>=6.2.4,<8.0.0",
+        "pytest>=7.0.0,<9.0.0",
```

Comment:
Note: It requires > 7 now, since `pytest-base-url` also requires Pytest v7 with the recent versions.

Reply:
I don't see us bumping `pytest-base-url`, so I don't understand why does the same version force us to the new `pytest`.

Reply:
They changed the requirement between v2.0.0 and v2.1.0.